benchmark/tangurnis/Makefile

```diff
@@ -6,10 +6,11 @@ tala_c: tala_c/vel_8.csv tala_c/vel_16.csv tala_c/vel_32.csv tala_c/vel_64.csv t
 
 
 
-%.csv: ../../aspect ba/tan.prm tala/tan.prm tala_c/tan.prm
+%.csv: Makefile code/aspect ba/tan.prm tala/tan.prm tala_c/tan.prm
```

Comment:
@tjhei maybe you can comment on this?
I think I've addressed all other comments about the king and tan gurnis benchmarks [here](https://github.com/jdannberg/aspect/commits/density_approximations).
In particular, I cleaned up the code and also updated the comments in the input files.